tests/diff.rs

```diff
@@ -110,7 +110,7 @@ mod gas {
 					let rules = utils::rules::Set::default();
 
 					let module = elements::deserialize_buffer(input).expect("Failed to deserialize");
-					let instrumented = utils::inject_gas_counter(module, &rules, "env")
+					let instrumented = utils::inject_gas_counter(module, &rules, "env", utils::ProcessNames::No)
```

Comment:
You should add a function name to the fixtures and enable it here. Also you may need to call `parse_names` on the module.